setup/aix61/manualBootstrap.md

```diff
@@ -303,6 +303,10 @@ installp -d /tmp/i-files -L
 installp -d /tmp/i-files -a aixtools.python3
 ```
 
+Repeat the __pip__ and __tap2junit__ steps above but substitute __python3__ in the commands in place of
```

Comment:
I grepped the ansible side of things. Using ansible, I think tap2junit will be installed only once. I guess changing that is one more thing to figure out.

Reply:
As discussed at https://github.com/nodejs/node/pull/29451#issuecomment-535669218